### PR TITLE
Update trigger.j

### PR DESCRIPTION
**Pull request by WaterKnight**
TriggerExecute/TriggerExecuteWait/TriggerWaitOnSleeps

### CODE REVIEW

**Comment by Cokemonkey11** [comment]
LGTM

**Comment by lep** [comment]
Could you expand on `TriggerWaitOnSleeps` instead of just linking to 
another function? Also please put all native names in backticks.
Otherwise looks good to me. Thx

Am 28.07.2019 um 17:28 schrieb Water:

**Comment by WaterKnight** [comment]
@lep updated

File: trigger.j [hunk 1]
```diff
@@ -19,7 +19,9 @@ native IsTriggerEnabled takes trigger whichTrigger returns boolean
 
 
 /**
-See TriggerExecuteWait
+Marks the given trigger to wait/no longer wait for `TriggerSleepAction`s in sub trigger executions started via `TriggerExecuteWait`.
+Since this is an attribute of the execution rather than the trigger object, this affects but prospective runs of the given trigger,
```
Comment (Cokemonkey11) [inline trigger.j]: Better: `, this affects future runs of the given trigger, and not those already started`

Comment (WaterKnight) [reply trigger.j]: updated

**Comment by Cokemonkey11** [comment]
LGTM

**Comment by Cokemonkey11** [comment]
@lep closed this without merge?

**Comment by lep** [comment]
i rebased locally. just look at the commits.

**Comment by Cokemonkey11** [comment]
I mean github has a rebase option :P but ok

**Pull request closed without merging**